src/services/ProjectTokenHistoricalPriceService.ts

```diff
@@ -127,13 +127,11 @@ export class ProjectTokenHistoricalPriceService {
           }
         }
 
-        const timezoneOffset = new Date().getTimezoneOffset() * 60;
         const priceAtTimePoint = fundingTokenPricesUSD.filter(price => price.timestamp <= timestamp );
 
-
         if (todaysSwaps?.length) {
           returnArray.push({
-            time: timestamp - timezoneOffset + intervalSeconds/* Apply to the next interval in users timezone */,
+            time: timestamp + intervalSeconds/* Apply to the next interval in users timezone */,
```

Comment:
We should document this function, and include a description such as:  "The returned data points will be relative to the timezone of the LBPManager startTime and endTime, which, according to the LBPManager entity, are in the user's timezone.